modules/ossm-preparing-to-update-istio-control-plane-with-inplace.adoc

```diff
@@ -0,0 +1,72 @@
+// Module included in the following assemblies:
+// update/ossm-updating-openshift-service-mesh.adoc
+
+:_mod-docs-content-type: Procedure
+[id="preparing-to-update-istio-control-plane-with-inplace_{context}"]
+= Preparing to update Istio control plane with InPlace strategy
+
+You can configure the {istio} control plane and workloads for the `InPlace` update strategy. When using the `InPlace` strategy, the `IstioRevision` resource created by the Operator always uses the same name as the `{istio}` resource. 
+
+. Attach the workloads to a control plane deployed using the `InPlace` strategy by running the following command. The following example sets the `istio.io/rev` namespace label to the name of the `IstioRevision`.
++
+[source,terminal]
+----
+$ oc label namespace <namespace-name> istio.io/rev=<revision-name>
+----
+
+. Alternatively, apply the revision label to individual workloads by modifying the pod template in the `Deployment` resource.
++
+[source,yaml, subs="attributes,verbatim"]
+----
+apiVersion: apps/v1
+kind: Deployment
+spec:
+  template:
+    metadata:
+      labels:
+        istio.io/rev: <revision-name>
+----
+
+. If the revision name is `default`,attach the workloads to the revision by running the following command. IThe following example labels the namespace with `istio-injection: enabled` label.
++
+[source,terminal]
+----
+oc label namespace <namespace-name> istio-injection=enabled
+----
+
+. Deploy the {istio} control plane using the `InPlace` update strategy. The following example configuration creates an `{istio}` resource named `default` in the `istio-system` namespace:
++
+[source,yaml, subs="attributes,verbatim"]
+----
+apiVersion: sailoperator.io/v1
+kind: Istio
+metadata:
+  name: default
+spec:
+  namespace: istio-system
+  version: v1.24.2
+  updateStrategy:
+    type: InPlace
+----
+
+. Install the {istio} CNI plugin with the desired version. The following example configuration creates an `IstioCNI` resource named `default` in the `istio-cni` namespace:
++
+[source,yaml, subs="attributes,verbatim"]
+----
+apiVersion: sailoperator.io/v1
+kind: IstioCNI
+metadata:
+  name: default
+spec:
+  version: v1.24.2
+  namespace: istio-cni
+----
+
+. Configure application workloads to run in the cluster. For example, deploy the `bookinfo` application in the `bookinfo` namespace.
+
+. Label the `bookinfo` namespace to enable sidecar injection by running the following command:
```

Comment:
`oc create ns bookinfo`

`oc apply -f https://raw.githubusercontent.com/openshift-service-mesh/istio/release-1.24/samples/bookinfo/platform/kube/bookinfo.yaml -n bookinfo`

Add these 2 commands to this step, it will install the bookinfo pods in bookinfo ns